### ISSUE TITLE
V13.0.0-56: Cannot read _id of undefined

### ISSUE THREAD

**Issue by emreeren** [y-js/yjs#114]
It appears to go away if I decrease dependency to -55. I don't have a meaningful stack trace because of minification but If needed I can debug it in detail.

**Comment by emreeren** [comment]
Here is where I'm receiving the error message. `Id !== null` does not seem to capture `undefined`.

![image](https://user-images.githubusercontent.com/835381/39551681-a3e82a34-4e6e-11e8-8121-0dbc4287a19b.png)

**Comment by emreeren** [comment]
Well.. I noticed that part doesn't changed on -55. I found this relates with having different versions of yjs under separate npm packages. It works fine now.

**Comment by emreeren** [comment]
No I'm still receiving it. It happens when I enable logging and relates with `item._start` property. Is it something new? My `itemJson` does not contain `_start` probably because they built using previous versions. 

![image](https://user-images.githubusercontent.com/835381/39634670-19343fa2-4fc4-11e8-896b-098c7a75a7e4.png)

**Comment by dmonad** [comment]
Hey @emreeren 
The data shared between beta releases are not necessarily compatible with each other. In version *-56 I implemented a new garbage collection approach, the old versions can't handle the data produced by versions pre *-56. 

Can you upgrade to version *-60 and check if you can reproduce the issue?

I will find the issue if you give me steps to reproduce.

**Comment by emreeren** [comment]
Hello. I cloned yjs & y-websockets-client. Opened examples/chat/index.js file and added `localStorage.debug = 'y*';` line to enable debug logging. When I launched two browser pages, added few messages and refresh one of them I could see the error message. 

![image](https://user-images.githubusercontent.com/835381/39819656-a537325c-53ac-11e8-922b-dc561415dec0.png)

I guess that relates with `Item` class.  `_start` property didn't initialized with `null` value like other properties.

https://github.com/y-js/yjs/blob/c0e630b635768b7923cdea71f12512cc05c18dc7/src/Struct/Item.js#L54

So _start remains `undefined` and it leads the error in `logID` function as it strictly checks the `null` value.

https://github.com/y-js/yjs/blob/c0e630b635768b7923cdea71f12512cc05c18dc7/src/MessageHandler/messageToString.js#L34

Initializing _start parameter or changing null equality check inside logID function as `!= null` both solves the issue but I'm not sure which one is the correct solution :)

Thanks.
